client/src/client.rs

```diff
@@ -345,7 +345,10 @@ impl Client {
     ///
     /// # Errors
     /// Fails if generating signature fails
-    pub fn sign_transaction(&self, transaction: Transaction) -> Result<Transaction> {
```

Comment:
This is the place, where I decided to create this issue. Finally it's fixed, thanks!